webapp/config.py

```diff
@@ -24,3 +24,6 @@ class ConfigurationError(Exception):
     raise ConfigurationError("`WEBAPP` is not configured")
 
 WEBAPP_EXTRA_HEADERS = {}
+
+SEND_FILE_MAX_AGE_DEFAULT = 10*365*24*60*60
```

Comment:
Can you add a comment as to how long this is for?

Reply:
Done!